internal/wire/analyze.go

```diff
@@ -353,9 +353,11 @@ func buildProviderMap(fset *token.FileSet, hasher typeutil.Hasher, set *Provider
 		}
 		concrete := providerMap.At(b.Provided)
 		if concrete == nil {
-			pos := fset.Position(b.Pos)
-			typ := types.TypeString(b.Provided, nil)
-			ec.add(notePosition(pos, fmt.Errorf("no binding for %s", typ)))
+			setName := set.VarName
+			if setName == "" {
+				setName = "wire.Build"
```

Comment:
This isn't always the case: one can always write an anonymous `wire.NewSet` call in the middle of a larger expression.

Reply:
Hmm, I copied this from somewhere else (`bindingConflictError`, near the end of file). What do you suggest? "Anonymous wire.NewSet"?

Reply:
Perhaps just "provider set"? The line number should already convey where the problem is.

Reply:
Done.